web/modules/custom/farm_page/src/Controller/FarmPageController.php

```diff
@@ -90,11 +90,16 @@ public function myFarm($uid) {
       }
       else {
         $store = Store::load($store_id);
-        $products = $this->farmService->getFeatureProducts((int) $store_id);
-        $tags = $this->farmService->getTaxonomyTerm($store->get('field_tags')->getValue());
-        $store_images_field = $store->get('field_farm_image')->getValue();
-        $store_images_url = $this->farmService->getImagesUrlFromArray($store_images_field);
-        $full_information_products = $this->farmService->getProductsInfomation($products);
+        $featured_products = $this->farmService->getFeatureProducts((int) $store_id);
+        if (isset($store)) {
+          $tags = $this->farmService->getTaxonomyTerm($store->get('field_tags')->getValue());
```

Comment:
variables inside this check will be undefined if $store is not set. Therefore, warning will appear when passing to the theme. => Init variables outside of condition block.

Reply:
variables initialized